### DISCUSSION TITLE
Lisence question about "the client formally known as redis-cli"

### DISCUSSION THREAD

**Discussion by ranshid** [valkey-io/valkey#95] [General]
I see that we decided to switch redis-cli to valkey-cli, which is great!
My question is: is the cli under the definition of "Redis clients" in terms of the SSPLv1/RSALv2?
(same question goes for redis/valkey-benchmark)

**Comment by madolson** [comment]
It's unlikely that anyone would be using either the cli or the benchmark under server definition of SSPLv1 or RSALv2. I don't think that changes our position about the tooling though, we would like all the tooling to refer to Valkey.
